docs/send-data/kubernetes/troubleshoot-collection.md

```diff
@@ -395,6 +490,268 @@ kube-prometheus-stack:
 
 where `metadata.name` is the value from Argo Application manifest.
 
+### Check metrics content
+
+You can print metrics on stdout of metrics collector and metrics metadata, and validate if they are correct. It may happen that metrics are ingested, but with different metadata than you expect.
+
+In order to print them on stdout, two steps are required:
+
+1. Disable ingesting logs from metrics related pods. This is required to prevent logs ingest spike. Add the following configuration to `user-values.yaml`:
+
+   ```yaml
+   debug:
+     metrics:
+       metadata:
+         stopLogsIngestion: true
+       collector:
+         stopLogsIngestion: true
+   ```
+
+   Then update your collection and wait for all logs collector pods to be redeployed.
+
```

Comment:
```suggestion
1. Disable ingesting logs from metrics related pods. This is required to prevent logs ingest spike.
   - Add the following configuration to `user-values.yaml`:
   ```yaml
   debug:
     metrics:
       metadata:
         stopLogsIngestion: true
       collector:
         stopLogsIngestion: true
   ```
   - Then, update your collection and wait for all logs collector pods to be redeployed.
```